staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -290,6 +295,8 @@ func (r *crdHandler) updateCustomResourceDefinition(oldObj, newObj interface{})
 	r.customStorageLock.Lock()
 	defer r.customStorageLock.Unlock()
 
+	r.ec.queue.AddAfter(newCRD.Name, 1 * time.Millisecond)
```

Comment:
is there a difference to a zero delay?

Reply:
and here we would need the case distinction: if apiserver-count > 1, then use a higher number. And we need a good comment why this is necessary (about the race).

Reply:
This is still not addressed (apiserver-count part), but I've updated to use zero delay. No difference at all, it works and is more intuitive.